framework/include/executioners/EigenExecutionerBase.h

```diff
@@ -131,6 +131,8 @@ class EigenExecutionerBase : public Executioner
   FEProblemBase & _problem;
   MooseEigenSystem & _eigen_sys;
 
+  FEProblemSolve _feproblem_solve;
```

Comment:
We have to build a dummy solve object for this to-be-outdated executioner otherwise line search, etc. will not be set properly.